apps/dialtone-documentation/docs/.vuepress/baseComponents/TokenTable.vue

```diff
@@ -148,6 +159,26 @@ export default {
     getSecondValue (value) {
       return value.split(' / ')[1];
     },
+
+    onEnterRow (name) {
+      console.log('ENTER', name);
+      this.hoveredRow = name;
+    },
+
+    onLeaveRow (name) {
+      console.log('LEAVE', name);
+      this.hoveredRow = null;
+    },
+
+    showCopyButton (name) {
+      return this.hoveredRow === name;
+    },
   },
 };
 </script>
+
+<style scoped>
+  .copy-button {
+    position: absolute;
+  }
+</style>
```

Comment:
why not using `class="d-ps-absolute"`?

Reply:
Oh, this was not supposed to be committed. Removed it, thank you